plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -112,7 +112,7 @@ public void testNoColumnStats()
         assertQuery("SELECT c_str FROM no_column_stats WHERE c_int = 42", "VALUES 'foo'");
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
why?

Reply:
This is related to supporting all access modes. See  #11782 

I have added `delta.security = system` and that causes the metadata to be retrieved in `MetadataManager` (before the access control was set to SYSTEM),

https://github.com/mdesmet/trino/blob/1c1fb3021648803d5138948e13b93c93ec2cf93d/core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java#L2257-L2264

Reply:
please reenable the test. 

then let's think how to make it pass. maybe we don't need `delta.security = system` here.
or maybe we need to merge the other PR first.
IDK yet, but let's have the test remind us (ie be failing until we can fix it)

Reply:
This change is related to the other PR. I will remove that commit here, but as I said before it seems that more tests are failing because of the AC setup in the BaseConnectorTest.